Libplanet/Net/Swarm.cs

```diff
@@ -1337,18 +1340,37 @@ private void TransferTxs<T>(BlockChain<T> blockChain, GetTxs getTxs)
             where T : IAction, new()
         {
             _logger.Debug("Trying to transfer blocks...");
+
+            var blocks = new List<byte[]>();
+
             foreach (HashDigest<SHA256> hash in getData.BlockHashes)
             {
                 if (blockChain.Blocks.TryGetValue(hash, out Block<T> block))
                 {
-                    Message response = new Block(block.ToBencodex(true, true))
+                    var payload = block.ToBencodex(true, true);
+                    blocks.Add(payload);
+                }
+
+                if (blocks.Count == getData.ChunkSize)
+                {
+                    Message response = new Messages.Blocks(blocks.ToArray())
                     {
                         Identity = getData.Identity,
                     };
                     _replyQueue.Enqueue(response);
+                    blocks.Clear();
                 }
             }
 
+            if (blocks.Any())
+            {
+                Message response = new Messages.Blocks(blocks.ToArray())
```

Comment:
`response`'s type seems to be obvious.
```suggestion
                var response = new Messages.Blocks(blocks.ToArray())
```